_doc/rule.rst

```diff
@@ -198,27 +198,35 @@ If you want to assign a custom name, you can change the rule name easily by assi
 
 *rule_a.py*::
 
-    class ClassA(Rule):
+    import HABApp
+
+    class ClassA(HABApp.Rule):
         ...
 
+        def function_a(self):
+          ...
+
     ClassA()
 
 *rule_b.py*::
 
+    import HABApp
+    import typing
+
     if typing.TYPE_CHECKING:            # This is only here to allow
-        from .class_a import ClassA     # type hints for the IDE
+        from .rule_a import ClassA     # type hints for the IDE
```

Comment:
Could you align it with the comment in the line above?

Reply:
Sure. I somehow missed this.